tfe/resource_tfe_team_access.go

```diff
@@ -14,6 +14,9 @@ func resourceTFETeamAccess() *schema.Resource {
 		Create: resourceTFETeamAccessCreate,
 		Read:   resourceTFETeamAccessRead,
 		Delete: resourceTFETeamAccessDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
```

Comment:
```
resource "tfe_team_access" "test" {
  team_id      = "${tfe_team.test.id}"
  workspace_id = "${tfe_workspace.canary.id}"
  access       = "read"
}
```

ex: `tf import tfe_team_access.test "tws-N32FjQga6SCJ34v3"`

```
-/+ tfe_team_access.test (new resource required)
      id:                                     "tws-N32FjQga6SCJ34v3" => <computed> (forces new resource)
      access:                                 "read" => "read"
      team_id:                                "team-2WbNREnViCFK17hP" => "${tfe_team.test.id}" (forces new resource)
      workspace_id:                           "" => "hashicorp-v2/canary" (forces new resource)
```

This seems not work at all. I doubt it would even overwrite (delete/create) the correct record.

Reply:
Yeah, this one is again missing data to be able to retrieve the required info (organization and workspace in this case). So this one is also not possible to import.